documentation/Deploy-MSBuild.md

```diff
@@ -0,0 +1,23 @@
+# Deploying Just-Built MSBuild
+
+## Visual Studio
+
+[Deploy-MSBuild](https://github.com/dotnet/msbuild/blob/deploy-msbuild/scripts/Deploy-MSBuild.ps1) is a way to conveniently take private bits and install them into Visual Studio (VS) for testing. To use it:
+
+- Build MSBuild with the changes you want using `build.cmd /p:CreateBootstrap=true`.
+- In an administrator powershell window, navigate to the msbuild folder.
+- Run `scripts\Deploy-MSBuild.ps1 -destination "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin" -configuration Debug`.
```

Comment:
I'm undecided about putting them inline like this, partially because there are reasons to use -configuration Release, but I like showing all the options inline. Maybe -configuration <configuration> then specify what that means below?

Reply:
Isn't that what I did? I specify parameters for both here and explain how they might need to be changed below.

Reply:
Markdown edited out part of my comment 😮 
`-configuration <configuration>` and explain what `<configuration>` means below is what I meant to say.

Reply:
Ahh. Yeah . . . that is much clearer :)